platform/features/clock/src/actions/PauseTimerAction.js

```diff
@@ -40,22 +37,39 @@ define(
          *        time (typically wrapping `Date.now`)
          * @param {ActionContext} context the context for this action
          */
-        function AbstractStartTimerAction(now, context) {
+        function PauseTimerAction(now, context) {
             this.domainObject = context.domainObject;
             this.now = now;
         }
 
-        AbstractStartTimerAction.prototype.perform = function () {
+        PauseTimerAction.appliesTo = function (context) {
+            var model =
+                (context.domainObject && context.domainObject.getModel()) ||
+                {};
+
+
+            // We show this variant for timers which have
+            // a target time, or is in a playing state.
+            return model.type === 'timer' &&
+                    model.timerState === 'started';
+        };
+
+        PauseTimerAction.prototype.perform = function () {
             var domainObject = this.domainObject,
                 now = this.now;
 
-            function setTimestamp(model) {
-                model.timestamp = now();
+            function setTimerState(model) {
+                model.timerState = 'paused';
+            }
+
+            function setPausedTime(model) {
+                model.pausedTime = now();
             }
 
-            return domainObject.useCapability('mutation', setTimestamp);
+            return domainObject.useCapability('mutation', setTimerState) &&
+                domainObject.useCapability('mutation', setPausedTime);
```

Comment:
Instead of having two mutations here, you could update both `timerState` and `pausedTime` in the same function, e.g.:

```
function updateTimerModel(model) {
    model.timerState = 'paused';
    model.pausedTime = now();
}

return domainObject.useCapability('mutation', updateTimerModel);
```

The same applies for a few of the other actions here.